### ISSUE TITLE
templates/f5-bigip-ctlr-deploy.yaml - securityContext wrong indentation

### ISSUE THREAD

**Issue by cirora** [F5Networks/k8s-bigip-ctlr#2422]
#### Setup Details
 CIS Version : 2.X      
 Build: f5networks/k8s-bigip-ctlr:latest       
 BIGIP Version: Big IP x.x.x      
 AS3 Version: 3.x      
 Agent Mode: AS3/CCCL       
 Orchestration: K8S/OSCP      
 Orchestration Version:       
 Pool Mode: Cluster/Nodeport      
 Additional Setup details: <Platform/CNI Plugins/ cluster nodes/ etc>

#### Description  
Looking at those rows, the indentation is wrong. runAsUser: {{ $securityContext.runAsUser }} and runAsUser: 1000  have different indentation. If I set runAsUser in values.yaml but not runAsGroup the yaml is not well formatted
```
       securityContext:
        {{- $securityContext := .Values.securityContext | default dict }}
        {{- if $securityContext.runAsUser }}
        runAsUser: {{ $securityContext.runAsUser }}
        {{- else }}
          runAsUser: 1000
        {{- end }}
        {{- $securityContext := .Values.securityContext | default dict }}
        {{- if $securityContext.runAsGroup }}
        runAsGroup: {{ $securityContext.runAsGroup }}
        {{- else }}
          runAsGroup: 1000
        {{- end }}

```

#### Steps To Reproduce
1) put in your values.yaml 
```
securityContext:
   runAsUser: 2000
```

#### Expected Result 
```
securityContext:
   runAsUser: 2000
   runAsGroup: 1000
   fsGroup: 1000
```

#### Actual Result  
```
securityContext:
   runAsUser: 2000
     runAsGroup: 1000
     fsGroup: 1000
```

#### Diagnostic Information
```buildoutcfg
<Configuration files, error messages, logs>
Note: Sanitize the data. For example, be mindful of IPs, ports, application names and URLs
Note: The following F5 article outlines the information required when opening an issue.
https://support.f5.com/csp/article/K60974137
```

**Comment by trinaths** [comment]
Created [CONTCNTR-3420] for internal tracking.

**Comment by vklohiya** [comment]
Fixed in PR https://github.com/F5Networks/k8s-bigip-ctlr/pull/2478. Fix will be available in CIS Charts 0.21 Release.
